sql/driver/doc.go

```diff
@@ -235,6 +238,45 @@ Querying rows:
 
 	SELECT __key, name FROM person WHERE age > 30
 
+2) "json" is a first-class SQL type with IMap and Kafka Connector support.
+
+Creating a mapping:
+
+	CREATE MAPPING person (
+		__key BIGINT,
+		this  JSON
+	)
+	TYPE IMAP
+	OPTIONS (
+		'keyFormat' = 'bigint',
+		'valueFormat' = 'json'
+	)
+
+Inserting rows:
+
+	// Use serialization.JSON type to specify a JSON value.
+	INSERT INTO person VALUES(100, serialization.JSON(fmt.Sprintf(`{"age":%d, "name":%s}`, 35, 'Jane Doe')))
+
+Querying rows:
+
+	// Use serialization.JSON type to scan JSON string.
+	q := fmt.Sprintf(`SELECT this FROM "%s" WHERE CAST(JSON_VALUE(this, '$.age') AS DOUBLE) > ?`, mapName)
+	rows, err := db.Query(q, minAge)
```

Comment:
It would be nice to add a comment about why the error handling is skipped, like: "error handling is omitted for to keep the example short".

Also, `defer rows.Close` is missing.

Reply:
Since the error handling is omitted, can you ignore the error:
```
rows, _ := db.Query(q, minAge)
```